src/index.js

```diff
@@ -39,7 +39,14 @@ if (
 }
 
 /* Warning if there are several instances of styled-components */
-if (process.env.NODE_ENV !== 'production' && typeof window !== 'undefined') {
+if (
+  process.env.NODE_ENV !== 'production' &&
+  process.env.NODE_ENV !== 'test' &&
+  typeof window !== 'undefined' &&
+  typeof navigator !== 'undefined' &&
+  navigator.userAgent.indexOf('Node.js') !== -1 &&
```

Comment:
This does the inverse of what it's supposed to do, I guess, since it includes only jsdom-environments, instead of excluding them

Reply:
woops! fixed